css/global.css

```diff
@@ -1,3 +1,3 @@
-.css-selector {
+.VideoPreviewPlayer, button[data-testid='__video-volume-btn'] {
```

Comment:
Grabs the preview player using its class and the button using the data-testid to remove from the DOM.